debian/rules

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
-VERSION := $(shell mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -v "\[")
+VERSION := $(shell awk 'f{print;f=0} /<artifactId>cloudstack/{f=1}' pom.xml| sed -e 's/<[^>]*>//g'  -e 's/^[ \t]*//')
```

Comment:
Well.. i'm actually building packages often because i'm working on it :) but since it's just a few seconds more, it's not a big deal.. i was more worried about that failure that i can't seem to find again... so, closing until i can pinpoint an actual issue with this in the jenkins build env :)